lib/hexo/validate_config.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = ctx => {
+  const { config, log } = ctx;
+
+  log.info('Validating config');
+
+  // Validation for config.url && config.root
+  if (typeof config.url !== 'string') {
+    throw new TypeError('_config.yml validation failed, "url" should be string!');
```

Comment:
I think `Invalid config detected. "url"...` is more suitable since `custom.yml` and `_config.json` are also supported.

Reply:
LGTM. I will update to that.